test/Test/Data/List.purs

```diff
@@ -236,6 +241,11 @@ testList = do
   assert $ (drop 2 (l [1, 2, 3])) == l [3]
   assert $ (drop 1 nil) == nil
 
+  log "drop should remove the specified number of items from the front of an list"
```

Comment:
drop**End** should remove the specified number of items from the **end** of an list